cypress/support/commands.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="cypress" />
+
+import '@testing-library/cypress/add-commands';
+
+Cypress.Commands.add('checkToastMessage', (type: 'success' | 'error', message: string) => {
+  const toastId = type === 'success' ? 'toast-success' : 'toast-error';
+
+  cy.get(`[id=${toastId}]`).should('contain', message);
+});
+
+Cypress.Commands.add('checkSpinner', (ariaLabel: string) => {
```

Comment:
Same suggestion to improve descriptiveness of the command:

```suggestion
Cypress.Commands.add('hasActiveSpinner', (ariaLabel: string) => {
```